.gitignore

```diff
@@ -31,3 +31,8 @@ build/
 
 ### VS Code ###
 .vscode/
+
+
+### My dev ###
+.env
+docker-compose.yaml
```

Comment:
why do you add `docker-compose.yaml` to gitignore?

Reply:
That file is elementary. I made it just for my convenience.